### PR TITLE
Moved operations G-L to ov namespace

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Depends on https://github.com/openvinotoolkit/openvino/pull/7341
 - Moved Parameter, Result to ov namespace

### CODE REVIEW

File: ngraph/core/src/op/hsigmoid.cpp [hunk 1]
```diff
@@ -15,7 +15,7 @@
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_RTTI_DEFINITION(op::v5::HSigmoid, "HSigmoid", 5);
+OPENVINO_RTTI_DEFINITION(op::v5::HSigmoid, "HSigmoid", 5);
```
Comment (mitruska) [inline ngraph/core/src/op/hsigmoid.cpp:18]:
```suggestion
OPENVINO_RTTI_DEFINITION(op::v5::HSigmoid, "HSigmoid", 5, op::util::UnaryElementwiseArithmetic);
```

File: ngraph/core/src/op/log.cpp [hunk 2]
```diff
@@ -12,7 +12,7 @@
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_RTTI_DEFINITION(op::v0::Log, "Log", 0);
+OPENVINO_RTTI_DEFINITION(op::v0::Log, "Log", 0);
```
Comment (mitruska) [inline ngraph/core/src/op/log.cpp:15]:
```suggestion
OPENVINO_RTTI_DEFINITION(op::v0::Log, "Log", 0, op::util::UnaryElementwiseArithmetic);
```

File: ngraph/core/src/op/loop.cpp [hunk 3]
```diff
@@ -15,7 +15,7 @@
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_RTTI_DEFINITION(op::v5::Loop, "Loop", 5);
+OPENVINO_RTTI_DEFINITION(op::v5::Loop, "Loop", 5);
```
Comment (mitruska) [inline ngraph/core/src/op/loop.cpp:18]:
```suggestion
OPENVINO_RTTI_DEFINITION(op::v5::Loop, "Loop", 5, op::util::SubGraphOp);
```

File: ngraph/core/src/op/hswish.cpp [hunk 4]
```diff
@@ -14,7 +14,7 @@
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_RTTI_DEFINITION(op::v4::HSwish, "HSwish", 4);
+OPENVINO_RTTI_DEFINITION(op::v4::HSwish, "HSwish", 4);
```
Comment (mitruska) [inline ngraph/core/src/op/hswish.cpp:17]:
```suggestion
OPENVINO_RTTI_DEFINITION(op::v4::HSwish, "HSwish", 4, op::util::UnaryElementwiseArithmetic);
```

File: ngraph/core/src/op/gru_cell.cpp [hunk 5]
```diff
@@ -14,7 +14,7 @@
 using namespace std;
 using namespace ngraph;
 
-constexpr NodeTypeInfo op::v3::GRUCell::type_info;
+OPENVINO_RTTI_DEFINITION(op::v3::GRUCell, "GRUCell", 1);
```
Comment (mitruska) [inline ngraph/core/src/op/gru_cell.cpp:17]:
```suggestion
OPENVINO_RTTI_DEFINITION(op::v3::GRUCell, "GRUCell", 1, op::util::RNNCellBase);
```

File: ngraph/core/include/openvino/op/lstm_sequence.hpp [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+///
+/// \note       It follows notation and equations defined as in ONNX standard:
+///             https://github.com/onnx/onnx/blob/master/docs/Operators.md#LSTM
+///
+/// \sa         LSTMCell, RNNCell, GRUCell
+///
+///
+class NGRAPH_API LSTMSequence : public util::RNNCellBase {
+public:
+    NGRAPH_RTTI_DECLARATION;
```
Comment (mitruska) [inline ngraph/core/include/openvino/op/lstm_sequence.hpp:151]:
```suggestion
    OPENVINO_RTTI_DECLARATION;
```

File: ngraph/core/include/openvino/op/lstm_sequence.hpp [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+///
+/// \note       It follows notation and equations defined as in ONNX standard:
+///             https://github.com/onnx/onnx/blob/master/docs/Operators.md#LSTM
+///
+/// \sa         LSTMCell, RNNCell, GRUCell
+///
+///
+class NGRAPH_API LSTMSequence : public Op {
+public:
+    NGRAPH_RTTI_DECLARATION;
```
Comment (mitruska) [inline ngraph/core/include/openvino/op/lstm_sequence.hpp:33]:
```suggestion
    OPENVINO_RTTI_DECLARATION;
```

File: ngraph/core/include/openvino/op/tensor_iterator.hpp [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/util/sub_graph_base.hpp"
+
+namespace ov {
+namespace op {
+namespace v0 {
+/// \brief  Iterate a body over tensors, accumulating into tensors.
+class NGRAPH_API TensorIterator : public op::util::SubGraphOp {
+public:
+    NGRAPH_RTTI_DECLARATION;
```
Comment (mitruska) [inline ngraph/core/include/openvino/op/tensor_iterator.hpp:19]:
```suggestion
    OPENVINO_RTTI_DEFINITION;
```

File: ngraph/core/include/openvino/op/tensor_iterator.hpp [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "openvino/core/function.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/util/sub_graph_base.hpp"
+
+namespace ov {
+namespace op {
+namespace v0 {
+/// \brief  Iterate a body over tensors, accumulating into tensors.
+class NGRAPH_API TensorIterator : public op::util::SubGraphOp {
```
Comment (mitruska) [inline ngraph/core/include/openvino/op/tensor_iterator.hpp:17]:
```suggestion
class OPENVINO_API TensorIterator : public op::util::SubGraphOp {
```

File: ngraph/core/include/openvino/op/lstm_sequence.hpp [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+///
+/// \brief      Class for lstm sequence node.
+///
+/// \note       It follows notation and equations defined as in ONNX standard:
+///             https://github.com/onnx/onnx/blob/master/docs/Operators.md#LSTM
+///
+/// \sa         LSTMCell, RNNCell, GRUCell
+///
+///
+class NGRAPH_API LSTMSequence : public util::RNNCellBase {
```
Comment (mitruska) [inline ngraph/core/include/openvino/op/lstm_sequence.hpp:149]:
```suggestion
class OPENVINO_API LSTMSequence : public util::RNNCellBase {
```

**Review by mitruska — APPROVED** [review]

File: ngraph/core/include/openvino/op/lstm_sequence.hpp (hunk 10 above)
Comment (ilyachur) [reply ngraph/core/include/openvino/op/lstm_sequence.hpp:149]: Will fix in the next PR

**Pull request merged**